package.json

```diff
@@ -2,7 +2,7 @@
   "name": "exercism",
   "private": true,
   "dependencies": {
-    "@codemirror/basic-setup": "^0.18.1",
+    "@codemirror/basic-setup": "^0.18.2",
```

Comment:
Adding `@codemirror/view` and specifying 0.18.13 explicitely here breaks Typescript quite badly.

Reply:
Fixed.